library/src/main/java/com/airbnb/android/airmapview/LeafletDivIcon.java

```diff
@@ -0,0 +1,40 @@
+package com.airbnb.android.airmapview;
+
+public class LeafletDivIcon {
```

Comment:
why "Icon"? DId mean "Element"?

Reply:
In [document](https://leafletjs.com/reference-1.3.0.html#divicon) of Leaflet, they call it `DivIcon`. So I use this name here.